From 771fd4a491fb250e7b18a020c9a36aae1997d779 Mon Sep 17 00:00:00 2001
From: Maulik Manvar <maulik.manvar@softwebsolutions.com>
Date: Mon, 14 Nov 2022 17:55:31 +0530
Subject: [PATCH] sm2s-rzg2ul : Override machine to sm2s-rzg2ul

-Added change to override smarc-rzg2ul with sm2s-rzg2ul

Signed-off-by: Maulik Manvar <maulik.manvar@softwebsolutions.com>
---
 meta/conf/machine/sm2s-rzg2ul.conf | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/meta/conf/machine/sm2s-rzg2ul.conf b/meta/conf/machine/sm2s-rzg2ul.conf
index 70bc3f6..e4fd642 100644
--- a/meta/conf/machine/sm2s-rzg2ul.conf
+++ b/meta/conf/machine/sm2s-rzg2ul.conf
@@ -2,6 +2,8 @@
 #@NAME: smarc-rzg2ul machine
 #@DESCRIPTION: RZ/G2UL SMARC platform
 
+MACHINEOVERRIDES =. "smarc-rzg2ul:"
+
 SOC_FAMILY = "r9a07g043u"
 require ../../meta-renesas.git/conf/machine/include/rzg2l-common.inc
 require ../../meta-renesas.git/conf/machine/include/cip-core.inc
@@ -12,7 +14,6 @@ TUNE_CONF ?= "../../meta-renesas.git/conf/machine/include/tune-cortexa55.inc"
 require ${TUNE_CONF}
 
 PREFERRED_PROVIDER_virtual/kernel = "linux-renesas"
-
 UBOOT_CONFIG ??= "smarc-rzg2ul"
 UBOOT_CONFIG[smarc-rzg2ul] = "smarc-rzg2ul_defconfig"
 
-- 
2.17.1

